nix/vast/default.nix

```diff
@@ -108,15 +82,29 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = lib.optional isStatic "pic";
 
-  doCheck = false;
+  doCheck = true;
+  # Concurrent testing is not possible because multiple suites reuse the same
+  # file paths.
+  enableParallelChecking = false;
+  preCheck = ''
+    export LD_LIBRARY_PATH=$PWD/libvast
+    export DYLD_LIBRARY_PATH=$PWD/libvast
+  '';
   checkTarget = "test";
 
   dontStrip = true;
 
   doInstallCheck = true;
-  installCheckInputs = [ py3 jq tcpdump ];
+  # We add those with an `overrideDerivation` in `overlay.nix`.
+  # installCheckInputs = [ py3 jq tcpdump ];
+  # TODO: Investigate why the disk monitor test fails in the build sandbox.
+  # TODO: Native plugins are currently broken with static builds.
   installCheckPhase = ''
-    python ../vast/integration/integration.py --app ${placeholder "out"}/bin/vast
+    python ../vast/integration/integration.py \
+      --app ${placeholder "out"}/bin/vast \
+      --disable "Disk Monitor"
+      --disable "Partition-local Stores"
+      --disable "Transforms"
```

Comment:
This won't run integration tests for lsvast and for plugins, which have separate integration test suites. Do we have a story for that already, or, is it possible to simply run the CMake build target `integration` here? We could add an option to CMake that skips the virtual env setup.

Reply:
For passing this through CMake we would have to make it possible to pass arbitrary options that get passed through to the integration testing script. This would be brittle because the `integration` target invokes it multiple times. I think its better to avoid that complexity and run the tests directly here. I will add support for integration testing of plugins in a follow up. lsvast can be checked with a second call directly here.